### DISCUSSION TITLE
Any idea why my text stroke would appear like that in server but not on web?

### DISCUSSION THREAD

**Discussion by yafkari** [fabricjs/fabric.js#10242] [Q&A]
Hey everyone, anyone already experienced difference in how an itext was rendered because of the stroke?

NodeJS
![image](https://github.com/user-attachments/assets/b89df57e-6119-42f1-bc76-dbacd889513b)

Web
![image](https://github.com/user-attachments/assets/88e5de24-6874-4c7f-a268-3ea314807ce9)

I have this options on the IText on both sides:
![image](https://github.com/user-attachments/assets/5c91f715-6c59-49f9-b2dc-5ae9b8c8ba9e)

It seems that on web when the stroke width is too large, it takes the whole space but not in node

(Web)
Width: 4
![image](https://github.com/user-attachments/assets/af3739e0-4b7d-428f-968b-34d57ba85597)

Width: 5
![image](https://github.com/user-attachments/assets/595e349d-81e7-4414-96c0-be06c1b0ecc5)

Width: 6
![image](https://github.com/user-attachments/assets/cc58cb4a-809c-46fb-b7fe-2bf2cb3c57d4)

This is exactly how i want it to react but seems like it doesn't seems to act like that on my backend.

I'm on both sides on Fabric 6.4.3

**Comment by asturur** [comment]
How does it render on firefox? can you post a reproducible use case? like a function that will output this in a png so someone can  look at it.
What font is it? Is it a custom font? what is the fontWeight?
Are both nodejs and the browser on the same machine and OS?
It can boil down to different font setting in the OS, those influence rendering for text.
